grails-app/controllers/com/unifina/controller/core/signalpath/CanvasController.groovy

```diff
@@ -95,18 +94,11 @@ class CanvasController {
 		render result as JSON
 	}
 
+	/**
+	 * @Deprecated See CanvasApiController#downloadCsv
+	 */
 	def downloadCsv() {
-		String fileName = System.getProperty("java.io.tmpdir") + File.separator + params.filename
-		File file = new File(fileName)
-		if (file.canRead()) {
-			FileInputStream fileInputStream = new FileInputStream(file)
-			response.setContentType("text/csv")
-			response.setHeader("Content-disposition", "attachment; filename="+file.name);
-			FileCopyUtils.copy(fileInputStream, response.outputStream)
-			fileInputStream.close()
-			file.delete()
-		}
-		else throw new FileNotFoundException("File not found: "+params.filename)
+		throw new FileNotFoundException("File not found")
```

Comment:
Is there a reason this method cannot be deleted?

Reply:
@harbu 
- There are references to downloadCsv() in .gsp files
- Unit tests pass even if method is deleted

I wasn't 100% sure whether deleting it breaks something so I left a stub.